facia-tool/public/js/models/collections/article.js

```diff
@@ -862,8 +865,9 @@ define([
 
                 self.updateEditorsDisplay();
             })
-            .catch(function() {
+            .catch(function(ex) {
                 self.meta.headline(undefined);
+                reportErrors(ex);
```

Comment:
BTW, `reportErrors` will rethrow the error unless you tell it not to: `reportErrors(error[, options, shouldRethrow])`